docker_config/dockerfiles/Dockerfile.default-gpu

```diff
@@ -32,7 +32,7 @@ RUN apt-get update && apt-get install -y \
 RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.6 1
 
-RUN curl https://bootstrap.pypa.io/pip/3.6/get-pip.py | python3.6
```

Comment:
@yuchenWYC are you sure these changes are needed (changes to pip url)? @jzwang43 just did the opposite of this to fix the docker builds a while ago -- see https://github.com/codalab/codalab-worksheets/pull/3992

Reply:
For this line I'm not 100% sure, since I tried a bunch of mixed things and kept the code that finally worked